packaging/nuget/nservicebus.host/EndpointConfig.cs

```diff
@@ -3,23 +3,15 @@ namespace rootnamespace
 {
     using NServiceBus;
 
-    /*
-		This class configures this endpoint as a Server. More information about how to configure the NServiceBus host
-		can be found here: http://particular.net/articles/the-nservicebus-host
-	*/
     public class EndpointConfig : IConfigureThisEndpoint
     {
         public void Customize(BusConfiguration configuration)
         {
-            // NServiceBus provides the following durable storage options
-            // To use RavenDB, install-package NServiceBus.RavenDB and then use configuration.UsePersistence<RavenDBPersistence>();
-            // To use SQLServer, install-package NServiceBus.NHibernate and then use configuration.UsePersistence<NHibernatePersistence>();
+            // NServiceBus provides multiple durable storage options
             
             // If you don't need a durable storage you can also use, configuration.UsePersistence<InMemoryPersistence>();
-            // more details on persistence can be found here: http://docs.particular.net/nservicebus/persistence-in-nservicebus
             
-            //Also note that you can mix and match storages to fit you specific needs. 
-            //http://docs.particular.net/nservicebus/persistence-order
+            //Also note that you can mix and match multiple storages to fit you specific needs. 
```

Comment:
Is this note even relevant? I think mixing and matching multiple storage options should be part of the documentation and along with it what the downsides are. Let's remove it.